### PR TITLE
fix: missing preview i18n

### PR DESCRIPTION
**Pull request by ycs77**
### Problem

I'm currently using the tutorialkit version v0.1.5, if added some i18n texts:

*docs/demo/src/content/tutorial/meta.md*
```md
---
type: tutorial
mainCommand: ['npm run dev', 'Starting http server']
prepareCommands:
  - ['npm install', 'Installing dependencies']

# add the i18n texts
i18n:
  prepareEnvironmentTitleText: '啟動環境'
  previewTitleText: '預覽'
  toggleTerminalButtonText: '切換終端機'
---
```

The i18n will work on webcontainer starting:

螢幕擷取畫面 2024-08-18 153215

But not working on webcontainer started:

螢幕擷取畫面 2024-08-18 155256

### PR Added
* Add missing `Toggle Terminal` i18n text in started `<Preview>`
* Add missing `Preview` i18n text

### CODE REVIEW

**Review by AriPerkkio — COMMENTED** [review]
The preview title is also configurable via `previews` option:

```yaml
---
type: tutorial
previews:
  - title: 預覽
    port: 5173
---
```

But it does make sense to be able to configure the default text via `i18n` as well.

**Comment by ycs77** [comment]
> The preview title is also configurable via `previews` option:

Oh, I see.

---

When I want to do i18n, I intuitively look for replaceable values ​​in the i18n options.

File: packages/types/src/schemas/i18n.ts
Comment (Nemikolh) [inline packages/types/src/schemas/i18n.ts:64]: I would be happy to merge this PR if we rename that text to `defaultPreviewTitleText`. What do you think @AriPerkkio ?

File: packages/types/src/schemas/i18n.ts
Comment (Nemikolh) [inline packages/types/src/schemas/i18n.ts:60]:
```suggestion
   * Text shown on top of the preview section when `previews[_].title` is not configured.
```

**Review by Nemikolh — COMMENTED** [review]
Really nice work! :star2: 

I think we can definitely land this. Thanks for the PR! :smiley:

File: packages/types/src/schemas/i18n.ts
Comment (AriPerkkio) [reply packages/types/src/schemas/i18n.ts:64]: I agree, `defaultPreviewTitleText` works better here. Let's use it.

**Review by AriPerkkio — CHANGES_REQUESTED** [review]
We should also mention this in documentation:

https://github.com/stackblitz/tutorialkit/blob/c7bad203045b702afda3176cece645bee4d4f6e3/docs/tutorialkit.dev/src/content/docs/reference/configuration.mdx?plain=1#L42-L81

File: packages/types/src/schemas/i18n.ts
Comment (ycs77) [reply packages/types/src/schemas/i18n.ts]: done

**Comment by ycs77** [comment]
Wait a minute while I look at why the test failed.

**Comment by AriPerkkio** [comment]
All tests are failing. We are pinning a dependency version of transitive dependency in test cases. Another dependency just updated and doesn't support that version. There's fix in https://github.com/stackblitz/tutorialkit/pull/261.

File: docs/tutorialkit.dev/src/content/docs/reference/configuration.mdx
Comment (Nemikolh) [inline docs/tutorialkit.dev/src/content/docs/reference/configuration.mdx:74]: Oh good catch! This comment is meant to be on top of `startWebContainerText?: string,`

**Review by Nemikolh — APPROVED** [review]
Looks good! I think the test failure is probably unrelated to this PR.

File: docs/tutorialkit.dev/src/content/docs/reference/configuration.mdx
Comment (ycs77) [reply docs/tutorialkit.dev/src/content/docs/reference/configuration.mdx:74]: So do we also need to update `i18n.ts`?

https://github.com/stackblitz/tutorialkit/blob/e45aa1523f82f71a406bdc63c98d7a0a7e277371/packages/types/src/schemas/i18n.ts#L25-L33

Comment (Nemikolh) [reply docs/tutorialkit.dev/src/content/docs/reference/configuration.mdx:74]: Oh good catch! Yes we should! :smiley:

Comment (ycs77) [reply docs/tutorialkit.dev/src/content/docs/reference/configuration.mdx:74]: done

**Comment by Nemikolh** [comment]
@ycs77 You have a lint error! Once fixed, we can merge your PR :raised_hands:

**Pull request merged**
